libpod/runtime_pod_linux.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 // NewPod makes a new, empty pod
-func (r *Runtime) NewPod(ctx context.Context, options ...PodCreateOption) (*Pod, error) {
+func (r *Runtime) NewPod(ctx context.Context, options ...PodCreateOption) (_ *Pod, err error) {
```

Comment:
I prefer to name the error Err, which eliminates confusion on whether or not err is used and forces does not have side effects affecting existing users of err.

Reply:
Fixed